### PR TITLE
Removed Unwanted Comments

### PR DESCRIPTION
**Pull request by arpit3018**
Please check the PR.
Thanks.

### CODE REVIEW

**Comment by monsij** [comment]
Hope that you have followed the contribution guidelines :-)

(Powered by [GitMate.io](https://gitmate.io))

File: pro1/users/models.py [hunk 1]
```diff
@@ -8,7 +8,7 @@ class Profile(models.Model):
     email_confirmed = models.BooleanField(default=False)
 
     def __str__(self):
-        return f'{self.user.username} Profile'
```
Comment (monsij) [inline pro1/users/models.py]: Restore this change as this is required for rendering

**Comment by monsij** [comment]
@arpit3018  : squash your commits into a single one. Give it a relevant commit message. It will then be ready to merge ⚡️

**Comment by monsij** [comment]
dq.sqlite3 should not be deleted

**Comment by monsij** [comment]
squash 3 commits into a single one

**Comment by arpit3018** [comment]
I am facing some issues while squashing. And db.sqlit3 is not deleted the chances includes me as a super user. It is been deleted. Would you please let me know about squash?

**Comment by monsij** [comment]
Didn't get the first part @arpit3018

**Comment by arpit3018** [comment]
In db.sqlite3 I added me as a super to remove this Change I have deleted that commit. And would You help me with squash coz I haven't done any yet?

**Comment by monsij** [comment]
That won't be a problem. Revert back the dq.sqlite3 file. For squashing look [here](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git)

**Comment by arpit3018** [comment]
Hey. Due to some issues I am re-forking this repo and will regenerate the PR.

**Pull request closed without merging**